lib/Interpreter/CppInterOp.cpp

```diff
@@ -3161,6 +3161,44 @@ namespace Cpp {
     return PI->getNameAsString();
   }
 
+  TCppFunction_t GetBinaryOperator(TCppScope_t scope, enum BinaryOperator op,
+                                   const std::string& lc,
+                                   const std::string& rc) {
+    Decl* D = static_cast<Decl*>(scope);
+    if (!D)
+      return nullptr;
+
+    DeclContext* DC = D->getDeclContext();
+    if (!DC)
+      return nullptr;
+
+    Scope* S = getSema().getScopeForContext(DC);
+    if (!S)
+      return nullptr;
+
+    clang::UnresolvedSet<8> lookup;
+
+    getSema().LookupBinOp(S, SourceLocation(), (clang::BinaryOperatorKind)op,
+                          lookup);
+
+    for (NamedDecl* x : lookup) {
+      if (auto* F = llvm::dyn_cast<Decl>(x)) {
```

Comment:
```suggestion
    for (NamedDecl* D : lookup) {
      if (auto* FD = llvm::dyn_cast<Decl>(D)) {
```

This is the common convention.